blog/pytensor_announcement.md

```diff
@@ -0,0 +1,56 @@
+(pytensor_announcement)=
+# PyMC is Forking Aesara to PyTensor
+
+:::{post} Nov 28, 2022
+:tags: announcement
+:category: news
+:author: PyMC developers
+:image: 0
+:exclude:
+
+The PyMC team would like to announce that we are forking the Aesara project.:::
```

Comment:
The closing colon fence needs to be on a line of its own.

Right now, this means everything until the next heading is considered to be inside the post directive. The text inside the post directive is what is used as excerpt in the listing of posts bydate, tag, category... And it can be excluded from the actual content of the blog, so it is only used there. As `:exclude:` is being used, all the first section is being excluded from the actual post and will appear only in the excerpt.

Reply:
@twiecki I think is related to this comment

Reply:
Yeah but I fixed that, maybe the html isn't updating?

Reply:
Oh, I pushed the wrong remote...